include/grpc/support/thd.h

```diff
@@ -66,9 +65,25 @@ int gpr_thd_new(gpr_thd_id *t, void (*thd_body)(void *arg), void *arg,
 /* Return a gpr_thd_options struct with all fields set to defaults. */
 gpr_thd_options gpr_thd_options_default(void);
 
+/* Set the thread to become detached on startup - this is the default. */
+void gpr_thd_options_set_detached(gpr_thd_options *options);
```

Comment:
No, I didn't realize that was the case; I've been mostly just using std::thread and not the gpr functions. So, I'm ok with this behavior as long as the default for the C++ wrapper class has joinable as a default.